mewarpx/mewarpx/utils_store/util.py

```diff
@@ -387,3 +386,37 @@ def plasma_Debye_length(T, n):
     return np.sqrt(
         constants.epsilon_0 * constants.kb_J * T / (n * constants.e**2)
     )
+
+def calc_timesteps(V_anode, CFL_factor, nx, nz, xmin, xmax, zmin, zmax, V_grid=5):
+        """Calculate timestep size based on grid data and CFL parameter
```

Comment:
Seeing how this has shaken out I think it is better to move this function to `mwxrun.py` and have it be a time counterpart to the `init_grid()` function. The reason is that it is unfortunate how would now have to call `mwxrun.init_grid(`xmin`, `xmax`, `zmin`, `zmax`, `nx`, `nz`)` with all the appropriate grid parameters and then again call `util.calc_timesteps()` with basically all the same parameters again. Instead we can have a function in `mwxrun.py` called `init_timestep` that takes either a fixed `DT` of a `CFL_factor` as input and if the `CFL_factor` is given it will calculate `DT` using the already defined grid quantities in `mwxrun`. Does that make sense?